averages.js

```diff
@@ -0,0 +1,41 @@
+function total(list) {
+  return list.reduce(function(sum, element) {
+    return sum + element;
+  }, 0);
+}
+
+function mean(list) {
+  return total(list)/(list.length);
+}
+
+function median(list) {
+  let sortedCollection = list.sort(function(a,b) {
+    return a - b;
+  });
+  let middleNum = sortedCollection.length / 2;
```

Comment:
Always reach for `const` first, and change to `let` if you find you need to reassign the variable.